ocs_ci/deployment/deployment.py

```diff
@@ -1626,8 +1648,23 @@ def deploy_with_external_mode(self):
         # enable secure connection mode for in-transit encryption
         if config.ENV_DATA.get("in_transit_encryption"):
             external_cluster.enable_secure_connection_mode()
-
-        setup_ceph_toolbox()
+        if config.DEPLOYMENT.get("multi_storagecluster"):
+            logger.info("not setting toolbox in multi-storagecluster")
+            health_check_namespace = constants.OPENSHIFT_STORAGE_EXTENDED_NAMESPACE
+        else:
+            setup_ceph_toolbox()
+            health_check_namespace = constants.OPENSHIFT_STORAGE_NAMESPACE
+        logger.info("Checking ceph health for external cluster")
+        try:
+            ceph_health_check(
+                namespace=health_check_namespace,
+                tries=30,
+                delay=10,
+            )
+        except CephHealthException as ex:
+            raise MultiStorageClusterExternalCephHealth(
```

Comment:
`MultiStorageClusterExternalCephHealth` is applicable only for the case of multi storage clusters. But here Ceph health check is done on any external mode cluster

Reply:
Now the whole logic of ceph_health_check is changed and other exception are used. Fixed it to have CephHealthExceptionExternal